CITATION

```diff
@@ -1,6 +1,6 @@
 Please cite as:
 
 Abigail Cabunoc and Sheldon McKay (eds): "Software Carpentry: Using
-Databases and SQL."  Version 2016.06, June 2016,
+Databases and SQL."  Version 2017.08, August 2017,
 https://github.com/swcarpentry/sql-novice-survey,
-10.5281/zenodo.57551.
+http://doi.org/10.5281/zenodo.838776
```

Comment:
Can you use https? (see [DOI handbook](https://www.doi.org/doi_handbook/3_Resolution.html#3.7.3))